packages/cadl-ranch-specs/http/type/model/visibility/main.tsp

```diff
@@ -30,6 +30,17 @@ model VisibilityModel {
   deleteProp: boolean;
 }
 
+/** RoundTrip model with readonly properties. */
+model ReadOnlyModel {
+  /** Required readonly nullable int list. */
+  @visibility("read")
+  requiredNullableIntList: int32[] | null;
```

Comment:
we will not be able to pass the case for Modular with the current design.

Reply:
which you cannot pass? you mean nullable?

Reply:
no, required readOnly case in input. is there a real case for it?

Reply:
It's roundtrip. But we can change it to optional.

Reply:
That would be nice to us, otherwise we probably won't be able to support it in a significant long time.

Reply:
fixed